fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1776,19 +1776,23 @@ public Status resetIdsForRestore(GlobalStateMgr globalStateMgr,
      * Post actions after restore. Rebuild the materialized view by using table name instead of table ids
      * because the table ids have changed since the restore.
      *
-     * @param db : the new database after restore.
      * @return : rebuild status, ok if success other error status.
      */
     @Override
-    public Status doAfterRestore(Database db, MvRestoreContext mvRestoreContext) throws DdlException {
-        super.doAfterRestore(db, mvRestoreContext);
+    public Status doAfterRestore(MvRestoreContext mvRestoreContext) throws DdlException {
```

Comment:
return value never used?

Reply:
Good catch! Ignore the status check in the final commit state because all backup data has already restored, and may `doAfterRestore` success later for all base tables are ready. 

But we can log the status later.